nsv13/code/modules/overmap/ai-skynet.dm

```diff
@@ -726,6 +726,9 @@ Adding tasks is easy! Just define a datum for it.
 				var/mob/dead/observer/C = pick(candidates)
 				target_ghost = C
 				var/obj/structure/overmap/GS = new selected_ship(target_location)
+				if(OM.current_system)
+					GS.current_system = OM.current_system
+					OM.current_system |= GS
```

Comment:
```suggestion
					OM.current_system.system_contents |= GS
```
I assume you want to add them to the system and not the current system variable of the overmap?

Reply:
Oh goodness, good catch.